packages/lesswrong/components/users/ReviewVotingPage.tsx

```diff
@@ -313,29 +327,34 @@ const ReviewVotingPage = ({classes}) => {
               <h2 className={classes.postHeader}>{expandedPost.title}</h2>
               <div className={classes.reason}>
                 <div className={classes.reasonTitle}>Anonymous Comments (optional)</div>
-                <CommentTextField 
-                  startValue={getVoteForPost(dbVotes, expandedPost._id)?.comment}  
+                <CommentTextField
+                  startValue={getVoteForPost(dbVotes, expandedPost._id)?.comment}
                   updateValue={(value) => submitVote({variables: {comment: value, postId: expandedPost._id}})}
                   postId={expandedPost._id}
                 />
               </div>
               <div className={classes.comments}>
-                <PostReviewsAndNominations 
-                  title="Nominations"
-                  terms={{view:"nominations2018", postId: expandedPost._id}} 
-                  post={expandedPost} 
-                />
-                <PostReviewsAndNominations 
-                  title="Reviews"
-                  terms={{view:"reviews2018", postId: expandedPost._id}} 
-                  post={expandedPost} 
-                />
+                <AnalyticsContext pageSectionContext="Nominations">
```

Comment:
Seems like it'd have been (weakly) better to put AnalyticsContext inside PostReviewsAndNominations, and set it's context equal to the "title" attribute.